website/docs/docs/contributing/oss-expectations.md

```diff
@@ -0,0 +1,104 @@
+
+### Initiative is everything
+
+Given that we, as maintainers, will not be able to resolve every bug or flesh out every feature request—if you, as a community member, want to see something change in dbt, the initiative rests with you.
+
+- If you open the bug report, it’s more likely to be identified.
+- If you open the feature request, it’s more likely to be discussed.
+- If you comment on the issue, engaging with ideas and relating it to your own experience, it’s more likely to be prioritized.
+- If you open a PR to fix an identified bug, it’s more likely to be fixed.
+- If you contribute the code for a well-understood feature, that feature is more likely to be in the next version.
+- If you review an existing PR, to confirm it solves a concrete problem for you, it’s more likely to be merged.
+
+Sometimes, this can feel like shouting into the void, especially if you aren’t met with an immediate response. We promise that there are dozens (if not hundreds) of folks who will read your comment, maintainers included. It all adds up to a real difference.
+
+# Practicalities
+
+As dbt OSS is growing in popularity, and dbt Labs has been growing in size, we’re working to involve new people in the responsibilities of OSS maintenance. We really appreciate your patience as our newest maintainers are learning and developing habits.
+
+## Discussions
+
+Discussions are a relatively new GitHub feature, and we really like them!
+
+A discussion is best suited to propose a Big Idea, such as brand-new capability in dbt Core, or a new section of the product docs. Anyone can open a discussion, add a comment to an existing one, or reply in a thread.
+
+What to expect from a new Discussion? Hopefully, comments from other members of the community, who like your idea or have their own ideas for how it could be improved. The most helpful comments are ones that describe the kinds of experiences users and readers should have. Unlike an **issue**, there is no specific code change that would “resolve” a Discussion.
+
+If, over the course of a discussion, we do manage to reach consensus on a way forward, we’ll open a new issue that references the discussion for context. That issue will connect desired outcomes to specific implementation details, as well as perceived limitations and open questions. It will serve as a formal proposal and request for comment.
+
+## Issues
+
+An issue could be a bug you’ve identified while using the product or reading the documentation. It could also be a specific idea you’ve had for how it could be better.
+
+### Best practices for issues
+
+- Issues are **not** for support / troubleshooting / debugging help. Please open a discussion instead (so other future users can search for it), or post in the dbt Slack Community for help understanding the problem you’re running into.
```

Comment:
Should these discussions always be opened on the docs.getdbt.com (to be renamed to more developer-friendly) repo? If so we may want to call that out @gwenwindflower

Reply:
We've also gotten a _few_ Q&A in the `dbt-core` repo; I agree it's best to have these Q&A centralized

Reply:
We've also gotten a small handful of Q&A/troubleshooting discussions in `dbt-core`. I agree it's best to centralize them in one place, where we can more easily search / point people / catch duplicates. FWIW it does look like [it's possible to transfer discussions](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions-in-your-repository#transferring-a-discussion)

Reply:
People inevitably will do this anyway. Do we want to outline what to expect when that happens so we can link them here?

Reply:
Good call. There is also a fine & fuzzy boundary between "I want help debugging" (discussion) and "I think I found a bug in dbt" (issue / bug report). There will be some judgment involved each time, and I expect we'll get better over time.

Do we feel right about this flow:
- Someone opens an issue that's actually a troubleshooting/support request
- The person (pre-)triaging [converts the issue into a Q&A discussion](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#converting-an-issue-to-a-discussion), and transfers the discussion to the central Q&A repository (whichever we decide, maybe this one)

Reply:
yeah that sounds reasonable to me. would we convert it after closing like when we discovered it's not a bug at the same time they fixed their issue?

Reply:
Converting the issue to a discussion also closes the issue. (Did I understand your question right?)

Reply:
Ah, yes, I think so! Something we should start doing going forward—once we know which repo we want to use for centralizing Q&A (is it this one?)